packages/kad-dht/src/constants.ts

```diff
@@ -15,15 +15,21 @@ export const MAX_RECORD_AGE = 36 * hour
 
 export const PROTOCOL = '/ipfs/kad/1.0.0'
 
-export const RECORD_KEY_PREFIX = '/dht/record'
+export const PROVIDERS_VALIDITY = 24 * hour
```

Comment:
A bit late to the party, but isn't this 48 hours 

See https://github.com/libp2p/go-libp2p-kad-dht/blob/18a758c0806da2525d7d4d7fecf8c2cf7cf12e0a/providers/providers_manager.go#L38

Reply:
It would seem so, can haz PR?